manifests/params.pp

```diff
@@ -53,6 +53,10 @@
       fail("xinetd: module does not support osfamily ${::osfamily}")
     }
   }
+  
+  $service_status = $::operatingsystem ? {
```

Comment:
@sulaweyo since this doesn't have anything but the default case I'd prefer to not have the selector here now, to be added in the future if the need arises. Thanks.